code/framework/schema/src/main/java/io/cattle/platform/schema/processor/AuthSchemaAdditionsPostProcessor.java

```diff
@@ -0,0 +1,61 @@
+package io.cattle.platform.schema.processor;
+
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.util.type.Priority;
+import io.github.ibuildthecloud.gdapi.factory.SchemaFactory;
+import io.github.ibuildthecloud.gdapi.factory.impl.AbstractSchemaPostProcessor;
+import io.github.ibuildthecloud.gdapi.factory.impl.SchemaPostProcessor;
+import io.github.ibuildthecloud.gdapi.model.Field;
+import io.github.ibuildthecloud.gdapi.model.impl.FieldImpl;
+import io.github.ibuildthecloud.gdapi.model.impl.SchemaImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.netflix.config.DynamicStringProperty;
+
+public class AuthSchemaAdditionsPostProcessor extends AbstractSchemaPostProcessor implements SchemaPostProcessor, Priority {
+
+    private static final DynamicStringProperty AUTH_SERVICE_EXTERNAL_ID_TYPES = ArchaiusUtil.getString("auth.service.external.id.types");
+    private static final List<String> EXTERNAL_ID_TYPES = new ArrayList<String>();
```

Comment:
Don't do this, it defeats the purpose of Dynamic properties.  Use `DynamicStringListProperty`